src/main/java/ch/iceguard/functions/MeasurementDao.java

```diff
@@ -0,0 +1,11 @@
+package ch.iceguard.functions;
+
+import java.util.List;
+
+public interface MeasurementDao {
```

Comment:
what about MeasurementDAO

Reply:
I'm more the camel case type of human :-)